src/kakarot/execution_context.cairo

```diff
@@ -181,7 +183,9 @@ namespace ExecutionContext {
         let (empty_events: model.Event*) = alloc();
         let (empty_create_contracts: felt*) = alloc();
         let (local revert_contract_state_dict_start) = default_dict_new(0);
-        tempvar revert_contract_state: model.RevertContractState* = new model.RevertContractState(revert_contract_state_dict_start, revert_contract_state_dict_start);
+        tempvar revert_contract_state: model.RevertContractState* = new model.RevertContractState(
```

Comment:
i updated my env by calling `make setup` and these syntax revisions was the result of calling `make format` with that updated env